app/assets/stylesheets/application.css

```diff
@@ -87,3 +83,23 @@ p{
   color: white;
   text-align: center;
 }
+
+.daily-header {
+  font-size: 30px;
+  text-align: center;
+  padding-top: 30px;
+  letter-spacing: 8px;
+}
+
+.daily-subheader {
+  font-size: 18px;
+  text-align: center;
+  padding-top: 30px;
+  letter-spacing: 5px;
+}
+
+.daily-subtitle {
+  font-size: 12px;
+  letter-spacing: 9px;
+  text-align: center;
+}
```

Comment:
css with page specific settings for daily page, but still following welcome page formatting